README.md

```diff
@@ -32,9 +32,7 @@ import (
 var cache cachego.Cache
 
 func init() {
-    cache = &cachego.Memcached{
-        memcached.New("localhost:11211")
-    }
+    cache = cachego.NewMemcached(memcached.New("localhost:11211"))
```

Comment:
Could you break the memcached.New and other dependencies in another line?

E.g.:

cache = cachego.NewMemcached(
    memcached.New("localhost:11211"),
)